model/src/w3swldmd.F90

```diff
@@ -1,5 +1,16 @@
+!> @file w3swldmd.F90
+!> @brief contains MODULE W3SWLDMD for swell dissipation source term.
+
 #include "w3macros.h"
 !/ ------------------------------------------------------------------- /
+!
+!> @author H. L. Tolman @date 21-Nov-2011
+!> @brief Source term module for swell dissipation.
+!>
+!> @details swell dissipation based on different
```

Comment:
Proper capitalization of sentences allows readers to easy see where one sentence ends, and another begins. Additionally, in general, we can assume some readers will have English as a second language. Proper capitalization helps them understand your documentation.

Reply:
Yes, I will edit this.